pvlib/spa.py

```diff
@@ -21,23 +21,15 @@ def nocompile(*args, **kwargs):
 
 if os.getenv('PVLIB_USE_NUMBA', '0') != '0':
```

Comment:
```suggestion
if os.getenv('PVLIB_USE_NUMBA') is not None:
```

Reply:
I think this change would require some corresponding changes in the part of `solarposition.py` that controls how `spa.py` is imported (by setting this environment variable to `'0'` or `'1'`), so I'm inclined to leave this as-is for now.  It will be addressed someday with #1060.

Reply:
I didn't realize that it could be defined as zero (I just assumed it would be undefined). Fine with me to just leave it.